src/common/utils/getHydratedPieces.js

```diff
@@ -1,27 +1,22 @@
-import {
-  batchFetch,
-  ITEM_STATUS,
-} from '@folio/stripes-acq-components';
+import { ITEM_STATUS } from '@folio/stripes-acq-components';
 
 export const getPieceStatusFromItem = (item) => {
   return item?.status?.name || ITEM_STATUS.undefined;
 };
 
-export function getHydratedPieces(pieces, mutatorRequests, mutatorItems) {
-  // TODO: fetch consortium context's data
-  const itemsIds = pieces.filter(({ itemId }) => itemId).map(({ itemId }) => itemId);
-  const requestsPromise = batchFetch(mutatorRequests, pieces, (piecesChunk) => {
-    const itemIdsQuery = piecesChunk
-      .filter(piece => piece.itemId)
-      .map(piece => `itemId==${piece.itemId}`)
-      .join(' or ');
-
-    return itemIdsQuery ? `(${itemIdsQuery}) and status="Open*"` : '';
-  });
+export function getHydratedPieces({
+  crossTenant,
+  fetchPieceItems,
+  fetchPieceRequests,
+  isConsortium,
+  pieces,
+}) {
+  const mutatorItems = fetchPieceItems({ pieces, crossTenant, isConsortium });
```

Comment:
Does `fetchPieceItems` accept the `isConsortium` option? [1](https://github.com/folio-org/ui-receiving/pull/548/files#diff-df665b67a35f8c88266b9ef2ed687247ead873aa70422c634fe6d0716e8486a0R67) -> [2](https://github.com/folio-org/ui-receiving/blob/12f89e31a76d46d495ed4e895827302c036a6497/src/common/hooks/usePaginatedPieces/hooks/usePieceItemsFetch.js#L11C9-L11C24)

And what is the purpose of `isConsortium` when `crossTenant` specifies requests for consortium endpoints?